edp/bounties.md

```diff
@@ -0,0 +1,40 @@
+# Bounties
+
+Our bounty program is an open invitation to everyone in the broader open source community to take part in the development of the Cere Network ecosystem. 
+
+The bounty program covers incentivized tasks from simply documentation fixes to building and maintaining core tooling, from validator nodes to SDKs and integrations. 
+
+Our priority is to ensure everyone in the community has equal opportunity to build their knowledge and skills from bottom up.
+
+## Types
+
+Bounties come in two forms: technical and non-technical. 
+
+From making transactions on the testnet to running simulations, testing, contributing to project code repositories, bounties come in many different types and sizes. 
+
+Some bounties are restricted in who can participate, but most will be designed so anyone, at anytime can jump right in and get started at the level they're most comfortable with from the start. 
+
+If you already possess the right knowledge already, we don't want to slow down your pace of innovation. We want you to work on the hardest, most valuable tasks you possible can! 
+
+Some bounties will be time constrained, but most will be available continuously for anyone to complete at their own convenience.
+
+The specifics of each bounty will be laid out in more detail as new activities are launched.  Make sure to join our community discord for the latest news!
+
+## Rewards
+
+All bounty rewards will be offered in the form of CERE tokens, unless otherwise noted.
+
+## Examples
+
+Some examples of the bounties include:
+
+- Technical
+  - preparing a deployable repository demonstrating a new use-case for (eg) [cere-ddc-sdk-js](https://github.com/Cerebellum-Network/cere-ddc-sdk-js)
+  - running a network simulation for validating that Cere Network slashing works as expected
+  - creating a new community maintained client SDK for a new language (eg, rust)
+  - adding new content to our [user-guide](https://docs.cere.network/ddc/developer-guide/examples)
+- Social
```

Comment:
Should we have the same bounties types (tech and non-tech or tech and social) in all documentation?